a8cteam51/build-processes-demo
# Adds JS build process to theme, and sets up an "extra-functionality" Plugin

#### Changes proposed in this Pull Request

* Adds a JS build process to the package theme - intended for Metaboxes etc. but could be used for anything
* Adds a JS build process to a secondary `mu-plugin`, `$-extra-functionality`, the idea of this plugin is to capture custom functionality that should be in plugins, like post types, etc. without polluting the blocks plugin.

#### Testing instructions



* Use the new `start/build:scripts:theme` and `start/build:scripts:plugin` scripts and ensure items are built from the `assets/js/src/` folder to the `assets/js/build/` folder.
